client/CMT.cpp

```diff
@@ -580,7 +581,7 @@ void playIntro()
 	}
 }
 
-#ifndef VCMI_IOS
+#if !defined(VCMI_IOS) && !defined(VCMI_ANDROID)

```

Comment:
Maybe we should add something like VCMI_MOBILE / VCMI_PHONE or something that will be defined for both ios and adroid? Code is full of "if android or ios" checks right now.

Reply:
indeed, we have a lot of `defined(VCMI_IOS) || defined(VCMI_ANDROID)` and inverted conditions. Will add `VCMI_MOBILE`.